src/routes.py

```diff
@@ -164,7 +179,7 @@ def login():
 @app.route("/settings", methods=['GET', 'POST'])
 def settings():
     pynder_session = db_util.load_pynder_session(session['username'])
-
+    profile = Profile(pynder_session._api.profile(), pynder_session._api)
```

Comment:
`profile` exista deja ca atribut in `pynder_session`.
functiile `form.update_profile_from_fields()` si `form.fill_fields_from_profile()` primesc doar `pynder_session` si isi aleg ele ce necesita.